li_add_local_image

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+#
+# Copyright (C) 2018 Layered Insight - All Rights Reserved
+# Unauthorized copying of this file, via any medium is strictly prohibited
+# Proprietary and confidential
+
+import argparse
+import sys
+import layint_api
+from layint_api.rest import ApiException
+from vyper import v
+from li_utils.envtools import env_setup
+from subprocess import call
+import uuid
+
+# Parse CLI
+parser = argparse.ArgumentParser()
+parser.add_argument("-v", "--verbose", help="Verbose logging", action="store_true")
+parser.add_argument("--name", help="Name of local image", required=True)
+args = parser.parse_args()
+
+env_setup()
+
+layint_api.configuration.api_key['Authorization'] = v.get("api_key")
+
+# Image API
+image_api = layint_api.ImageApi()
+image_api.api_client.host = v.get("api_host")
+
+# Create image.tar
+if len(args.name.split(":")) < 2:
+  print("Using default tag 'latest'")
+  args.name = args.name + ":latest"
+image_tar = "/tmp/"+str(uuid.uuid4())+".tar"
```

Comment:
Note the white space on this line and the line before it - try to keep the code looking similar.

Reply:
aah, got it